### PR TITLE
Create an Online platform

### PR DESCRIPTION
**Pull request by Azonkeu**
Hi 👋

- [ ] This pull request is to create the home and about section of an online platform where people can take online English lessons with native speakers.

- [ ] Here is the link to my loom video: https://www.loom.com/share/fb0871f755bf4b0bbaf8541cf1559518

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
-# Speak-Fluently
\ No newline at end of file
+# Speak-Fluently
+
+## Set up an Online Education Website
+
+In this project, we will build an online education website offering online English lessons to students from all over the world.
+
+#### Project requirements 
+
```
Comment (VanessaAoki) [inline README.md:8]: - [ ] [OPTIONAL]: To make this README file even more professional, you should consider adding a screenshot of your project.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,559 @@
+const mainTeacherBlock = document.createElement('div');
```
Comment (VanessaAoki) [inline script.js:46]: - [ ] I can see that you put a lot of effort into creating this script, but according to the project requirement, you should use a JavaScript variable with the data about the speakers and use it when the page loads. In order to do this, you should do the following:
- [ ] The list of speakers used to generate the HTML is assigned to a variable as an array or object.
- [ ] The list of speakers contains a speaker object for each that holds their specific data, for instance:
```
const speakerList = [ { name: "Speaker Name",... } ]
```
- [ ] The object contains all the information needed to generate the dynamic element, at least the name, role, description, and image URL about the speaker. For example:
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```
- [ ] There is a function that iterates through the speaker's list on window load and creates HTML elements with the data from each object. A good approach here would be using the [map function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).

File: about.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+const hamburgerone = document.getElementById('hamone');
+hamburgerone.addEventListener('click', () => {
+  document.getElementById('iconlistone').style.width = '100%';
+});
+
+const closeNavone = document.getElementById('closeNavone');
+closeNavone.addEventListener('click', () => {
```
Comment (VanessaAoki) [inline about.js:7]: - [ ] Right now, your close button is broken, you should consider fixing it. This is how it looks right now:

image

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @Azonkeu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Amazing video description 🎥 
- [x] Beautiful design and awesome theme choice 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @Azonkeu 👋  

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

- [ ] On mobile screen, your index page does not have a footer but both of your pages should have the same navbar and footer.


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**